deploy_heroku.py

```diff
@@ -171,12 +173,12 @@ def get_git_push_spec():
 
 # Add the redis add-ons (the heroku one needs a config flag)
 run_heroku_command(["addons:create", "heroku-redis:hobby-dev",
-                    "--maxmemory_policy", "allkeys-lru"])
+                    "--maxmemory_policy", "allkeys-lru", "--timeout", "1800"])
```

Comment:
@czlee Just a heads up that we should keep an eye on socket timeout behaviours. Daphne defines its own `websocket_timeout` and channels defines a `group_expiry`. I'm not super clear how these all interact.

Reply:
I see, to be honest I wasn't sure, I was just copying the app.json manifest.